RoutingStrategies.xaml.cs

```diff
@@ -0,0 +1,40 @@
+using Microsoft.Maui.Controls;
```

Comment:
According to [the docs](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/map?view=net-maui-8.0), "The [Map](https://learn.microsoft.com/en-us/dotnet/api/microsoft.maui.controls.maps.map) control is provided by the [Microsoft.Maui.Controls.Maps NuGet package](https://www.nuget.org/packages/Microsoft.Maui.Controls.Maps/), which should be added to your .NET MAUI app project."

Did you do this? The .csproj file was not part of this PR, which suggests that you didn't add it to the project, and might explain the issue with map not being found. But then I don't know how this would've compiled ...